api/controller.py

```diff
@@ -552,12 +520,11 @@ def authenticated_patron_root_lane(self):
             return patron
 
         policy = Configuration.root_lane_policy()
-        lane_info = policy.get(patron.external_type)
-        if lane_info is None:
+        lane_identifier = policy.get(patron.external_type)
```

Comment:
This seems redundant given Lane.root_for_patron_type. Should this look for a Lane that's the root for the patron's type so we don't need the policy?

Reply:
Yes, exactly. This is the payoff for adding root_for_patron_type to the database -- we don't need to put the policy into JSON anymore.